### PR TITLE
Bugfix FXIOS-7552 [v120] Fixed console warnings for layout constraints

### PR DESCRIPTION
**Pull request by imsanchit**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7552)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/16797)

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: BrowserKit/Sources/ComponentLibrary/CollapsibleCardView.swift [hunk 1]
```diff
@@ -167,24 +173,15 @@ public class CollapsibleCardView: ShadowCardView, UIGestureRecognizerDelegate {
 
         headerView.addSubview(titleLabel)
```
Comment (razvanlitianu) [inline BrowserKit/Sources/ComponentLibrary/CollapsibleCardView.swift:174]: hey @imsanchit There's a separate issue for CollapsibleCardView, could you move it in a separate PR? https://github.com/mozilla-mobile/firefox-ios/issues/16798

Comment (imsanchit) [reply BrowserKit/Sources/ComponentLibrary/CollapsibleCardView.swift]: Thanks @razvanlitianu for pointing this out, I have removed the code for CollapsibleCardView to separate PR.

File: Client/Frontend/Fakespot/Views/FakespotAdjustRatingView.swift [hunk 2]
```diff
@@ -119,7 +111,6 @@ class FakespotAdjustRatingView: UIView, Notifiable, ThemeApplicable {
 
     private func adjustLayout() {
         let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
-        starRatingHeightConstraint?.constant = min(UIFontMetrics.default.scaledValue(for: UX.starSize), UX.starMaxSize)
```
Comment (razvanlitianu) [inline Client/Frontend/Fakespot/Views/FakespotAdjustRatingView.swift:114]: Removing the height constraint causes the stars to not grow with accessibility sizes. You can reproduce by increasing the font from Xcode or by adding Text Size on the control center on the iPhone.


| | |
|--------|--------|
| image | image |

Comment (imsanchit) [reply Client/Frontend/Fakespot/Views/FakespotAdjustRatingView.swift]: Thanks @razvanlitianu for pointing this out. Have updated the PR, please have a look.

**Review by thatswinnie — APPROVED** [review]

**Review by razvanlitianu — APPROVED** [review]
